docs/installation/micro-rdk-dev.md

```diff
@@ -251,6 +256,22 @@ To obtain this:
 
 {{% snippet "secret-share.md" %}}
 
+### Add your machine cloud credentials as a file in your project
```

Comment:
@mattjperez found this necessary but seems like a bug-- how does micro-RDK team want to address?

Reply:
made a [ticket](https://viam.atlassian.net/browse/RSDK-8623?atlOrigin=eyJpIjoiY2VlOGQ2N2M0MjVhNDkwYzhkMWJlNzgwMDM2NDM4MWEiLCJwIjoiaiJ9) and PR. I reworded the prompt to say machine cloud config like what's in UI, but will need to see what others think.

Reply:
fix merged this morning. this section should no longer be needed, however I'd include checking the viam.json file in general troubleshooting steps

Reply:
awesome, sounds great thanks!